src/extension.ts

```diff
@@ -31,7 +31,7 @@ export async function activate(context: vscode.ExtensionContext, isRestart = fal
     disposables.push(outputChannel);
 
     config.init();
-    rootLogger.init(LogLevel[config.logLevel], config.logPath, config.outputToConsole, outputChannel);
+    rootLogger.init(vscode.env.logLevel, config.logPath, config.outputToConsole, outputChannel);
```

Comment:
this isn't what we want. we want the channel-local log level (which is persisted by choosing `Set As Default`)

```suggestion
    rootLogger.init(outputChannel.logLevel, config.logPath, config.outputToConsole, outputChannel);
```

Reply:
Oh interesting. I'll have to play with this a bit later.

Reply:
Huh, I actually think that given we don't modify the output channel log level, these two are actually equivalent. That said, I'll change it, as it is definitely clearer to do it that way.

From the [docs](https://code.visualstudio.com/api/references/vscode-api#LogLevel):

> The current log level of the channel. Defaults to [editor log level](https://code.visualstudio.com/api/references/vscode-api#env.logLevel).